File: AEDS/AEDS2/src/exercicios/estruturas/arvore/c/no.c
#include "no.h"
#include <stdio.h>
#include <stdlib.h>

No *novoNo(int elemento){
	No* no = (No*) malloc(sizeof(No));
	no->elemento = elemento;
	no-> dir = NULL;
	no->esq = NULL;
	return no;
}

File: AEDS/AEDS2/src/exercicios/estruturas/arvore/java/No.java
package exercicios.estruturas.arvore.java;

public class No {
    int elemento;
    No esq,dir;

    public No(int e){
        elemento =e;
        esq = dir = null;
    }
    No(){
        this.elemento = 0;
        esq = dir = null;
    }
}
